scribus/scribusdoc.cpp

```diff
@@ -4517,7 +4520,13 @@ class UsedGlyphsPainter: public TextLayoutPainter
 		}
 	}
 
-	void drawGlyphOutline(const GlyphLayout gl, bool, bool) { drawGlyph(gl, false); }
+	void drawGlyphOutline(const GlyphLayout gl, bool, bool)
+	{
+		if (gl.glyph >= ScFace::CONTROL_GLYPHS)
+			return;
+
+		drawGlyph(gl, false);
+	}
```

Comment:
You don’t need the check here since this method just calls `drawGlyph` which already does the check.